src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/process/DefaultProcessRunner.java

```diff
@@ -107,8 +110,10 @@ public void run(String[] command) throws ProcessRunnerException {
         processBuilder.environment().putAll(environment);
       }
 
+      processBuilder.directory(workingDirectory);
```

Comment:
This should really be setDirectory or setWorkingDirectory but that's probably another PR